### DISCUSSION TITLE
Codenewbie logo as a homepage link

### DISCUSSION THREAD

**Discussion by Mr-Destructive** [forem/forem#15823] [Feature Requests]
## In the navigation bar, the logo should be added to access the homepage while reading an article, viewing the dashboard, etc. 

![image](https://user-images.githubusercontent.com/40317114/146677041-0c43582f-5e05-4025-87b9-3344560d0902.png)

**Comment by cmgorton** [comment]
Hey @Mr-Destructive I am not sure I am following your request. When I go to CodeNewbie either while logged in or logged out the CodeNewbie logo is at the top and can be clicked while reading an article, etc. to navigate back to the homepage.
![CodeNewbie logo](https://user-images.githubusercontent.com/21039864/147119273-566c3332-fbe6-4d14-bf83-a24308b96162.png)
 
 
 Is that what you meant? Or where you asking for something else?

**Reply by Mr-Destructive** [reply]
Sorry, there actually was not available when I logged in.  I tries in incognito and another browser, but it was not rendering at that time, so I thought it would be nice to have one.
Ya it's there now :+1:  Thanks

**Reply by cmgorton** [reply]
No worries! It might have been a bug on our part then that has now been fixed. Thanks for opening this discussion.
